src/router/routes.js

```diff
@@ -198,6 +198,20 @@ module.exports = [
 			},
 		]
 	},
+	{
+		path: '/lend-by-category/loan-to-women',
+		// eslint-disable-next-line no-unused-vars
+		redirect: to => {
+			return { path: '/lend-by-category/women' };
+		}
```

Comment:
The redirect for both of these routes can just be a string, since it's a static path that is being redirected to. That means that the eslint-disable-next-line comment won't be needed either:
```
{
	path: '/lend-by-category/loans-to-women',
	redirect: '/lend-by-category/women'
}
```